src/test/cypress/tsconfig.json

```diff
@@ -6,6 +6,6 @@
         "ignoreDeprecations": "5.0",
         "target": "es2019",
         "sourceMap": false,
-        "types": ["cypress", "@4tw/cypress-drag-drop", "node", "cypress-wait-until", "cypress-file-upload"]
-    }
+        "types": ["cypress", "@4tw/cypress-drag-drop", "node", "cypress-wait-until", "cypress-file-upload"],
+    },
```

Comment:
I'm pretty sure json files don't support trailing commas. Also suggested by the codacy alert.